dump-db/inc/dump.js

```diff
@@ -1,6 +1,6 @@
 const fs = require("fs");
 const sanitize = require("sanitize-filename");
-const sql = require('./sql.js');
+const sql = require('./sql');
 const decryptService = require('./decrypt.js');
```

Comment:
@ProfDoof , it's a mistake on my side, it should resolve to `dump-db/inc/dump.js` and probably I did a global find & replace with a few exceptions and forgot about `dump-db`.

Nevertheless, I think this should not cause an actual impact since `require` should tolerate both `.js` and `.ts`. I've changed it back for consistency, though.